### PR TITLE
Project Manager: Update template dialog position after window resize

### PR DESCRIPTION
**Pull request by lupoDharkael**

### CODE REVIEW

File: editor/project_manager.cpp [hunk 1]
```diff
@@ -1753,6 +1753,10 @@ void ProjectManager::_notification(int p_what) {
 
 			Engine::get_singleton()->set_editor_hint(false);
 		} break;
+		case NOTIFICATION_RESIZED: {
+
+			open_templates->popup_centered_minsize();
```
Comment (akien-mga) [inline editor/project_manager.cpp]: That seems weird, it will popup whenever you resize the window, even if it was not shown previously, no?

Comment (lupoDharkael) [reply editor/project_manager.cpp]: oh that's true xD I was testing in a system without projects so I didn't notice it was still showing after having any project. I'll fix it.

**Comment by akien-mga** [comment]
Thanks!

**Comment by Calinou** [comment]
Should we do this for other editor popups in general?

Would it be possible to abstract it so we don't have to do it for every popup?

**Comment by akien-mga** [comment]
Definitely. I merged in a moment of weakness to get `pr_count--`, but it's indeed only a workaround and a more systemic fix would be better.

**Pull request merged**